cmake/external/leveldb.cmake

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Is there anyway to avoid depending on this?

Reply:
I will delete this if I found a way to avoid depending on this.

Reply:
What is the reason for depending on it now?

Reply:
https://github.com/brpc/brpc/blob/master/CMakeLists.txt#L116